common/src/Exceptions.h

```diff
@@ -40,13 +40,13 @@ class ExceptionStream : public Exception {
     ExceptionStream() noexcept :
```

Comment:
We should aim to phase out/remove this, right? Seems overkill to bring in the heavy dependency on stringstream/iostream just for throwing a TB exception type, which in most cases won't need to do any fancy formatting to generate the exception string

Reply:
The thing is, it's certainly very convenient to have this. I tried a few replacements, but all of them were less nice to use. But I agree that it should go, I suppose we have to use stringstream to format the messages at the use sites instead.

Reply:
It's done now.